setup.py

```diff
@@ -33,7 +33,7 @@
     install_requirements = [
         "pandas",
         "pyyaml>=5.1",
-        "vivarium",
+        "vivarium<1.1.0",
```

Comment:
We have to update version at the same time we fix PRL?

Reply:
We'll pin a lower bound of 1.1.0 when we next release pseudopeople.